Sources/TestSupport/misc.swift

```diff
@@ -309,6 +310,10 @@ public func waitForFile(_ path: AbsolutePath) -> Bool {
     return false
 }
 
+public func target() -> String {
+    return try! Destination.hostDestination().target
```

Comment:
This will be computed on every access. I think we can put this in an extension as a static property and cache it.

```swift
private let _cachedHostTarget = try! Destination.hostDestination().target
extension Destination {
    /// Returns the host target.
    static var hostTarget: String {
        return _cachedHostTarget
    }
}
```